src/linker/Linker.Steps/MarkStep.cs

```diff
@@ -3905,32 +3909,36 @@ void MarkMethodsFromReflectionCall (ref ReflectionPatternContext reflectionConte
 			void MarkPropertiesFromReflectionCall (ref ReflectionPatternContext reflectionContext, TypeDefinition declaringType, string name, bool staticOnly = false)
```

Comment:
There are now similar `MarkPropertiesOnTypeHierarchy` and also for fields `MarkFieldsOnTypeHierarchy` - maybe we should merge the two - if possible.